rosidl_generator_py/test/test_interfaces.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
```

Comment:
If this file is executable it won't be used by nose. If it is not it doesn't need a shebang line.